hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/BlockInfo.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
IntelliJ doesn't find any uses of this class?

Reply:
It was WIP this week because the EIP155 support was prioritized for the 0.47 release